.prettierrc.json

```diff
@@ -1,3 +1,30 @@
 {
-  "parser": "html"
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "useTabs": true,
+  "trailingComma": "es5",
+  "bracketSpacing": true,
+  "arrowParens": "always",
+  "printWidth": 80,
+  "overrides": [
```

Comment:
According to https://prettier.io/docs/en/options.html#parser we shouldn't necessarily have to set the parser explicitly (I'm not sure why it was specified before).  Do you think we could just remove the "overrides" section?

Reply:
Yes, It's unnecessary to include the overrides section. Removed the overrides section to make it more flexible and cleaner.